src/with-component.js

```diff
@@ -2,9 +2,10 @@ import { render } from 'preact';
 import { HTMLElement } from './util';
 import { withProps } from './with-props';
 import { withRender } from './with-render';
+import { withUnique } from './with-unique';
 
 export function withComponent (Base = HTMLElement) {
-  return class extends withRender(withProps(Base)) {
+  return class extends withUnique(withRender(withProps(Base))) {
```

Comment:
Actually, Canary is acceptable: 
<img width="987" alt="stable" src="https://cloud.githubusercontent.com/assets/136556/24355962/c73119f4-1343-11e7-9285-e14030cdbd7d.png">
<img width="1002" alt="canary" src="https://cloud.githubusercontent.com/assets/136556/24355967/c9b84c60-1343-11e7-874c-4a2e2bc78075.png">